packages/widget-chat/src/components/activity-item-post/index.js

```diff
@@ -2,22 +2,55 @@ import React, {PropTypes} from 'react';
 import classNames from 'classnames';
 
 import Avatar from '../avatar';
-import ActivityItemPostActions from '../activity-item-post-actions';
+import ActivityItemPostAction from '../activity-item-post-action';
 import styles from './styles.css';
 
+import {ICON_TYPE_DELETE, ICON_TYPE_FLAGGED_OUTLINE} from '../icon';
 
 export default function ActivityItemPost(props) {
+  function handleOnDelete() {
+    const {id, onActivityDelete} = props;
+    onActivityDelete(id);
+  }
+
+  function handleOnFlag() {
+    const {id, onActivityFlag} = props;
+    onActivityFlag(id);
+  }
+
+  function wrapActionItem(action, highlight) { // eslint-disable-line react/no-multi-comp
```

Comment:
Yea, I considered making this another component as well but either got lazy or decided it was overly complicated for something so basic. I can't remember which right now, haha